frontend/website/src/Careers.re

```diff
@@ -65,107 +214,33 @@ let make = (~jobOpenings, _) => {
-                      {str("Collaboration")}
-                    </h2>
-                    <p className="w-65 mt0 mb0 lh-copy">
-                      {str(
-                         {js|The problems we face are novel and challenging and we take them on as a\u00A0team.|js},
-                       )}
-                    </p>
-                  </div>
-                </div>
-                <div className="db dn-ns">
-                  <div className="mt45">
-                    <h2
-                      className="fw5 mt0 mb0 ml15 f2 ocean f2-ns tracked-tightly">
-                      {str("Collaboration")}
-                    </h2>
-                    <p className="mt3 mb0 ml15 lh-copy">
-                      {str(
-                         {js|The problems we face are novel and challenging and we take them on as a\u00A0team.|js},
-                       )}
-                    </p>
-                  </div>
-                </div>
-              </div>
-              <div>
-                <div className="dn db-ns">
-                  <div className="flex justify-between mt45">
-                    <h2
-                      className="pr2 fw5 mt0 mb0 ml15 f2 ocean f2-ns tracked-tightly">
-                      {str("Inclusion")}
-                    </h2>
-                    <p className="w-65 mt0 mb0 lh-copy">
-                      {str(
-                         {js|We're working on technologies with the potential to reimagine social structures. We believe it's important to incorporate diverse perspectives from conception through\u00A0realization.|js},
-                       )}
-                    </p>
-                  </div>
-                </div>
-                <div className="db dn-ns">
-                  <div className="mt45">
-                    <h2
-                      className="fw5 mt0 mb0 ml15 f2 ocean f2-ns tracked-tightly">
-                      {str("Inclusion")}
-                    </h2>
-                    <p className="mt3 mb0 ml15 lh-copy">
-                      {str(
-                         {js|We're working on technologies with the potential to reimagine social structures. We believe it's important to incorporate diverse perspectives from conception through\u00A0realization.|js},
-                       )}
-                    </p>
-                  </div>
-                </div>
-              </div>
+              <HeadingItem title="Open Source">
+                {ReasonReact.string(
+                   {js|We passionately believe in the open-source philosophy, and make our software free for the entire world to\u00A0use.|js},
+                 )}
+                <a
+                  href="/static/code.html"
+                  className="dodgerblue fw5 no-underline hover-link nowrap">
+                  {ReasonReact.string({js|Take a look →|js})}
+                </a>
+              </HeadingItem>
+              <HeadingItem title="Collaboration">
+                {ReasonReact.string(
```

Comment:
oh I liked `str` here, but whichever you prefer

Reply:
Don't really need it right now I think